src/material/schematics/ng-add/index.spec.ts

```diff
@@ -418,3 +418,33 @@ describe('ng-add schematic', () => {
     });
   });
 });
+
+describe('ng-add schematic - library project', () => {
+  let runner: SchematicTestRunner;
+  let libraryTree: Tree;
+  let errorOutput: string[];
+  let warnOutput: string[];
+
+  beforeEach(async () => {
+    runner = new SchematicTestRunner('schematics', require.resolve('../collection.json'));
+    libraryTree = await createTestLibrary(runner);
+
+    errorOutput = [];
+    warnOutput = [];
+    runner.logger.subscribe(e => {
+      if (e.level === 'error') {
+        errorOutput.push(e.message);
+      } else if (e.level === 'warn') {
+        warnOutput.push(e.message);
+      }
+    });
+  });
+
+  it('should warn that the material has beend added to a library project', async () => {
```

Comment:
```suggestion
  it('should warn that the material has been added to a library project', async () => {
```

Reply:
I'd propose `should warn if a library project is targeted`.